src/rustdoc/membership.md

```diff
@@ -0,0 +1,116 @@
+
+Being promoted to member implies a number of privileges:
+
+- Members have `r+` (approve a pull request) privileges and can do reviews (they are expected to
+  use those powers appropriately, as discussed previously). They also have access to control
+  perf/rustc-timer and other similar bots. See the documentation for `bors` and `r+`
+  [here](https://rustc-dev-guide.rust-lang.org/contributing.html#r-1).
+
+  Tip: some baseline rules around bors permissions are: don't do a `try` build unless you have
+  done a check for malicious code first and don't `r+` unless you are reasonably confident that
+  you can effectively review the code in question.
+- Rustdoc team members are members of the Rust organization so they can modify labels and be
+  assigned to issues.
+- Members become a part of the `rust-lang/rustdoc` team on GitHub, so that they receive pings
+  when people are looking to address the team as a whole.
+- Members are listed on the [rust-lang.org web page].
+
+It also implies some obligations (in some cases, optional obligations):
+
+- Members may take part in various other [maintainer activities] to help the team.
+- Members are held to a higher standard than ordinary folk when it comes to the [Code of
+  Conduct][CoC].
+
+[rust-lang.org web page]: https://www.rust-lang.org/governance/teams/dev-tools#team-rustdoc
+
+## What it means to be a rustdoc member
+Once you're a member of the rustdoc team, a number of events will happen:
+
+- You will gain access to a private Zulip stream, where internal discussions happen.
+- You will also be subscribed to the `all@rust-lang.org` and `rustdoc@rust-lang.org` mailing lists.
+  See [this file](https://github.com/rust-lang/team/blob/HEAD/teams/all.toml) to check how
+  subscriptions to mailing lists work. Both are very low-volume mailing list (maybe a few emails per
+  year). About `all@rust-lang.org`: it's a way to communicate things to all contributors. We will
+  not send you spam from this address.
+
+## Roles in the rustdoc team
+
+Rustdoc has multiple different areas and team members are not working on all of them. Currently
+we have three main areas:
+
+- Front-end: Everything related to HTML/CSS/JS
+- JSON backend: Work on the `--output-format=json` backend.
+- Internals: The internals of rustdoc: interacting with the compiler, doctests, generating
+  rustdoc internal code representation, parsing command line arguments, lints, etc.
+
+These groups are NOT full-fledge teams, and as such, to be part of any of these groups, you need to
+be a member of the rustdoc team.
+
+For now, only the front-end group has an official status in the
+[team repository](https://github.com/rust-lang/team) and is called `rustdoc-frontend`. If a rustdoc
+team member is interested to be part of this group, they can ask to be added into it.
+
+Being part of the front-end group doesn't change your rustdoc team membership. However
+you will be assigned for reviews on front-end pull requests and on front-end FCPs.
```

Comment:
two thoughts about this:
- i don't think being a team member should strongly imply that you have to do reviews (although certainly we can encourage it). again, i think the minimum is that people respond to FCPs.
- "doesn't change your rustdoc team membership" is a little unclear - i think you meant "you don't have to be on t-rustdoc to be on t-rustdoc-frontend"? can you write that explicitly?

Reply:
> * i don't think being a team member should strongly imply that you have to do reviews (although certainly we can encourage it). again, i think the minimum is that people respond to FCPs. vice-versa, it seems fine for people on t-rustdoc to review frontend prs even if they're not on t-rustdoc-frontend.

Indeed, clarifying text.

> * "doesn't change your rustdoc team membership" is a little unclear - i think you meant "you don't have to be on t-rustdoc to be on t-rustdoc-frontend"? can you write that explicitly?

No I really meant that you need to be part of the rustdoc team to be able to have the t-rustdoc-frontend role. Gonna clarify it.

Reply:
> No I really meant that you need to be part of the rustdoc team to be able to have the t-rustdoc-frontend role.

oh, interesting. my impression was that these "roles" are so that people can be involved with one part of rustdoc without having to be involved with all parts. if we require someone to be on t-rustdoc to have a role, they'd still be responsible for all FCP's, not only the FCP's related to their role. Is that intentional?

Reply:
If you're not part of any role, then you're not part of any FCP either, but that'd be strange, if someone is not part of any role, I don't think they should be on the team, or maybe there is a role missing and we can always add a new one if we think it makes sense. I listed 3 roles for the team:
* front-end
* JSON back-end
* "everything else" (so that includes the HTML back-end, "cleaning" (rustc's to rustdoc's AST), rustdoc lints and more generally rustdoc internals)

Normally it covers all our current cases. If we want to make a general FCP, we can tag all the roles.

Reply:
No problem! Do you think it needs clarifications? If you misunderstood this part, others might as well but not sure which part led to this confusion.

Reply:
Marking as resolved then.